<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdExtraIOWindow</class>
 <widget class="QMainWindow" name="QxrdExtraIOWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_DetectorsTab">
       <attribute name="title">
        <string>Detectors</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="m_DetectorsTable"/>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="m_RestartButton">
            <property name="toolTip">
             <string>Restart Sync</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/system-reboot.png</normaloff>:/images/system-reboot.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_SyncInfoButton">
            <property name="toolTip">
             <string>Sync Info Browser</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/help-about.png</normaloff>:/images/help-about.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>368</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="m_NewDetectorButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_DeleteDetectorButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="qxrdresources.qrc">
              <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_OutputsTab">
       <attribute name="title">
        <string>Outputs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="m_NewOutputButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="qxrdresources.qrc">
                <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="m_DeleteOutputButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="qxrdresources.qrc">
                <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTableView" name="m_OutputsTable"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_OutputWaveformsTab">
       <attribute name="title">
        <string>Output Waveforms</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QxrdExtraOutputsPlotWidget" name="m_OutputWaveforms" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_InputsTab">
       <attribute name="title">
        <string>Inputs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="4">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0" colspan="2">
           <widget class="QTableView" name="m_InputsTable">
            <property name="toolTip">
             <string>Channel Information</string>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="m_DeleteInputButton">
          <property name="toolTip">
           <string>Remove channel</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="m_NewInputButton">
          <property name="toolTip">
           <string>Add a new channel</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_InputWaveformsTab">
       <attribute name="title">
        <string>Input Waveforms</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QxrdExtraInputsPlotWidget" name="m_InputWaveforms" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_FileMenu">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionJunk"/>
   </widget>
   <widget class="QMenu" name="m_EditMenu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionJunk_2"/>
   </widget>
   <widget class="QMenu" name="m_WindowMenu">
    <property name="title">
     <string>W&amp;indow</string>
    </property>
    <addaction name="actionJunk_3"/>
   </widget>
   <addaction name="m_FileMenu"/>
   <addaction name="m_EditMenu"/>
   <addaction name="m_WindowMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionJunk">
   <property name="text">
    <string>&amp;Junk</string>
   </property>
  </action>
  <action name="actionJunk_2">
   <property name="text">
    <string>&amp;Junk</string>
   </property>
  </action>
  <action name="actionJunk_3">
   <property name="text">
    <string>&amp;Junk</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxrdExtraInputsPlotWidget</class>
   <extends>QWidget</extends>
   <header>qxrdextrainputsplotwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QxrdExtraOutputsPlotWidget</class>
   <extends>QWidget</extends>
   <header>qxrdextraoutputsplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qxrdresources.qrc"/>
 </resources>
 <connections/>
</ui>
